scout/adapter/mongo/panel.py

```diff
@@ -61,16 +63,13 @@ def load_omim_panel(self, api_key, institute=None):
 
         LOG.info("Setting version to %s", version)
 
-        try:
-            mim_files = fetch_mim_files(api_key=api_key, genemap2=True, mim2genes=True)
-        except Exception as err:
-            raise err
-
         date_string = None
         # Get the correct date when omim files where released
-        for line in mim_files["genemap2"]:
+        for line in genemap2_lines:
             if "Generated" in line:
                 date_string = line.split(":")[-1].lstrip().rstrip()
```

Comment:
lstrip().rstrip() isn't the same as just strip()?

Reply:
Haha yes I don't understand why not strip is used..

Reply:
Fixed!